### DISCUSSION TITLE
17 gennaio 2024 B - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#94] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/960c04a4-f5c5-4be8-b240-81534927bd0d)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-23 120641](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/5a33c396-c959-4717-82bc-efbb1f1c27f1)
![Immagine 2024-01-23 120657](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/55b6e17c-53b9-4058-bcb3-18cdc648a45a)

**Reply by daromacarlo** [reply]
3c) #RM = (2^10) * 11 * 5 giusto?

**Reply by MatteoGeusa** [reply]
Perché nel terzo rigo del punto 3a hai scritto che per calcolare i max record per blocco del file indice dividi 50 000 per 45+4 e non 2048? sicuro sia giusto?

**Reply by daromacarlo** [reply]
Sono un po' arrugginito sull'argomento, anche se rifacessi l'esercizio sarebbe poco attendibile la risposta 🙃

**Comment by Frisbon** [comment]
250k = NR

350 = RS

45 = KS

2048 = BS

4 = PS

RpB = <(BS/RS) = <(2048/350) = 5

TotBlockMain = >(NR/RpB) = >(250000/5) = 50k

RpBI = <(BS/PS + KS) = <(2048/4+45) = 41 

TotBlockIndex = >(TotBlockMain/RpBI) = >(50.000/41) = 1220

AccessTime = >(log_2(1220) + 1) = 12

**Comment by cafollaludovica** [comment]
![IMG_0897](https://github.com/user-attachments/assets/7cb08e61-a7cc-4c14-bd67-dd3c6ff9072d)
